### DISCUSSION TITLE
Manage different names for a person

### DISCUSSION THREAD

**Discussion by newyorker123** [Kareadita/Kavita#3315] [Ideas]
### Idea Description

An artist may have different names or names in different languages. For now, I use parentheses and a full-width comma to include all the names, i.e., `平本アキラ(平本AKIRA，平本彰)`. But that's tedious to maintain.

Now that the People Pages are coming, maybe we can have a more advanced way to manage this. We can relate all the names, i.e., `平本アキラ`, `平本AKIRA`, and `平本彰`, to the same person and choose one of the names to be displayed on the detail page, regardless of which name is actually in the metadata.

In this way, we can enjoy the following conveniences:

1. When we add a new series with one of the three names, the system will recognize it as the same writer.
2. When we search for one of the names, all the works associated with all three names will be displayed.

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Alias support is 100% planned. I already implemented this whole person architecture (as a way to ensure it would work well) in another project which also has alias support. 

My current thoughts is a merge system which when merging 2 people, will take the 2nd person and make them an alias of the first. As ComicInfo is limited, aliasing will be required to get the best experience so the next scan works as expected.
